frontend/tasks/server.js

```diff
@@ -2,12 +2,14 @@ const gulp = require('gulp');
 const config = require('../config.js');
 
 const connect = require('gulp-connect');
+const historyApiFallback = require('connect-history-api-fallback');
```

Comment:
Ah, yeah docs would be nice, but there are none. So, this is actually something that needs fixing or else the static deploy will result in a 404.

New paths need to be added to [this list over here in the pages task](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/pages.js#L25)